### PR TITLE
Handle konnectors with AppCollection

### PR DESCRIPTION
**Pull request by CPatchane**
For now we just to fetch all konnectors from `/konnectors` endpoint.

### CODE REVIEW

File: packages/cozy-stack-client/src/KonnectorCollection.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  }
+}
+
+class KonnectorCollection {
+  constructor(stackClient) {
+    this.stackClient = stackClient
+  }
+
+  /**
+   * Lists all apps, without filters.
```
Comment (gregorylegarec) [inline packages/cozy-stack-client/src/KonnectorCollection.js]: s/apps/konnectors

File: packages/cozy-stack-client/src/__tests__/KonnectorCollection.spec.js [hunk 2]
```diff
@@ -0,0 +1,378 @@
+jest.mock('../CozyStackClient')
+
+import CozyStackClient from '../CozyStackClient'
+import KonnectorCollection from '../KonnectorCollection'
+
+const ALL_RESPONSE_FIXTURE = {
```
Comment (gregorylegarec) [inline packages/cozy-stack-client/src/__tests__/KonnectorCollection.spec.js]: To improve lisibility maybe the fixtures could be in an external file, what do you think ?

File: docs/api/cozy-stack-client.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 <dl>
 <dt><a href="#AppCollection">AppCollection</a></dt>
-<dd><p>Implements <code>DocumentCollection</code> API along with specific methods for <code>io.cozy.apps</code>.</p>
+<dd><p>Implements its own methods specific to io.cozy.apps</p>
```
Comment (ptbrowne) [inline docs/api/cozy-stack-client.md]:
```suggestion
<dd><p>Implements methods specific to io.cozy.apps</p>
```

File: packages/cozy-stack-client/src/KonnectorCollection.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
+import { normalizeDoc } from './DocumentCollection'
+import { uri } from './utils'
+
+export const KONNECTORS_DOCTYPE = 'io.cozy.konnectors'
+
+export const normalizeApp = konnector => {
```
Comment (ptbrowne) [inline packages/cozy-stack-client/src/KonnectorCollection.js]: normalizeKonn ? Has it been copy pasted ? Can it be shared ?

File: packages/cozy-stack-client/src/KonnectorCollection.js [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+    const path = uri`/konnectors/`
+    const resp = await this.stackClient.fetchJSON('GET', path)
+    return {
+      data: resp.data.map(konnector => normalizeApp(konnector)),
+      meta: {
+        count: resp.meta.count
+      },
+      skip: 0,
+      next: false
+    }
```
Comment (ptbrowne) [inline packages/cozy-stack-client/src/KonnectorCollection.js]: I think AppCollection and KonnectorCollection are very similar ? Why don't they share code ?

Comment (CPatchane) [reply packages/cozy-stack-client/src/KonnectorCollection.js]: You're right, let met try something to use the same codebase here 👍

**Comment by CPatchane** [comment]
@ptbrowne I used the same AppCollection for both `io.cozy.apps` and `io.cozy.konnectors` with the last commit, what do you think?

**Comment by ptbrowne** [comment]
LGTM

**Pull request merged**